students/BrianNisonger/Lesson8/assignment/inventory.py

```diff
@@ -0,0 +1,34 @@
+import csv
+import logging
+
+
+def add_furniture(*args):
+    with open(args[0], mode="a") as customer_file:
+        customer_writer = csv.writer(customer_file)
+        try:
+            customer_writer.writerow(
+                [args[1], args[2], args[3], f'{args[4]:.2f}'])
+        except IndexError:
+            logging.error("Missing values")
+            raise (IndexError)
+        except ValueError:
+            logging.error("Not a number")
+            raise (ValueError)
+    return None
```

Comment:
Not quite what they had in mind, but such piss poor instructions, how could you know? The idea was that you actually had all five things as inputs (rather than *args), and then in the single_customer you could use the add_furniture to loop through and add each line from the one csv file to the other. You would use partial so that you wouldn't have to add the customer name and the file each time. It doesn't really save you much, since it would be one line, but it was to demonstrate how to use partial to change the signature of a function. You got the closure right, though, and that is much more important than partial, which you may or may not ever use again. :)